### ISSUE TITLE
[Feature][Config UI] Config UI needs to support Gitee

### ISSUE THREAD

**Issue by yumengwang03** [apache/incubator-devlake#3320]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Like other data sources that Config UI already supports, we need to design the Connetion page and the Blueprint transformation page for Gitee.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by jinxinhe** [comment]
![image](https://user-images.githubusercontent.com/114125035/195110241-169e43b2-6fc8-4445-a23f-ac3d0731e146.png)
[https://www.figma.com/file/Eux4Dhorxp7M0TwzOnL6Yh/Config-UI-for-V1.0?node-id=1999%3A10071](url)

**Comment by mintsweet** [comment]
I'll take it.

**Comment by e2corporation** [comment]
@mintsweet Don't setup the **Issue Tracking** or **Code Review** sections, as we will need to extract those from JIRA transformations as mentioned to shared components, which I wanted @likyh to accomplish with TAPD setup. Also this depends on #3373 before formally starting a PR

**Comment by Startrekzky** [comment]
On hold

**Comment by yumengwang03** [comment]
The prototype needs to be updated.
